encode_dash.py

```diff
@@ -143,10 +144,10 @@ class AVCSD:
 
 class AVCHD:
     m_profile = "high"
-    m_level = "40"
+    m_level = "52"
```

Comment:
Not sure, see https://docs.google.com/spreadsheets/d/1hxbqBdJEEdVIDEkpjZ8f5kvbat_9VGxwFP77AXA_0Ao/edit?gid=0#gid=0

Reply:
Good catch. I rolled back the changes, these were unintended. The PR currently focuses on the HEVC content generation. I'll take another look at AVC.